### PR TITLE
Create page for bt_smarthub device tracker

### PR DESCRIPTION
**Pull request by jxwolstenholme**
**Description:**

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+---
```
Comment (frenck) [inline source/_components/device_tracker.bt_smarthub.markdown:14]: missing `ha_release`.

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+To use a BT Smart Hub router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+    host: 192.168.1.254
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/device_tracker.bt_smarthub.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- **host** (*Optional*): The IP address of your router, Default: 192.168.1.254.
+
+See the [device tracker component page](/components/device_tracker/) for instructions how to configure the people to be tracked.
```
Comment (frenck) [inline source/_components/device_tracker.bt_smarthub.markdown]: `for instructions how` -> `for instructions on how`

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+This platform offers presence detection by looking at connected devices to a [BT Smart Hub](https://en.wikipedia.org/wiki/BT_Smart_Hub) based router.
+This router is sometimes referred to as the BT Home Hub 6.
+
+To use a BT Smart Hub router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+    host: 192.168.1.254
```
Comment (frenck) [inline source/_components/device_tracker.bt_smarthub.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
@jxwolstenholme I've reviewed your PR, and left some comments. Could you please take a look? 👍

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+ha_release: "0.79.4"
```
Comment (frenck) [inline source/_components/device_tracker.bt_smarthub.markdown]: New features are never released in a minor release. So please set it to the next major release "0.80"

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/device_tracker.bt_smarthub.markdown (hunk 5 above)
Comment (jxwolstenholme) [reply source/_components/device_tracker.bt_smarthub.markdown]: Thanks

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for updating this PR!

✅ Looks good. Can be merged as soon as the parent PR gets merged in.

Happy Hacktoberfest 🎉 

../Frenck

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+This platform offers presence detection by looking at connected devices to a [BT Smart Hub](https://en.wikipedia.org/wiki/BT_Smart_Hub) based router.
+This router is sometimes referred to as the BT Home Hub 6.
+
+To use a BT Smart Hub router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+
```
Comment (tjorim) [inline source/_components/device_tracker.bt_smarthub.markdown]: Remove empty line here but add one between ``` on the next line and the configuration-tag.

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+ha_release: "0.80"
+---
+
+
```
Comment (tjorim) [inline source/_components/device_tracker.bt_smarthub.markdown]: Limit empty line to 1

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+```
+{% configuration %}
+host:
+  description: The IP address of your router 
+  Default: 192.168.1.254
+  required: false
+  type: string
+{% endconfiguration %}
+
+
```
Comment (tjorim) [inline source/_components/device_tracker.bt_smarthub.markdown]: Limit empty line to 1

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+    
+```
+{% configuration %}
+host:
+  description: The IP address of your router 
+  Default: 192.168.1.254
```
Comment (tjorim) [inline source/_components/device_tracker.bt_smarthub.markdown]: `default` should not have a capital letter

**Review by tjorim — CHANGES_REQUESTED** [review]
Nice first PR here, @jxwolstenholme, I left some feedback for you.

⚠️ The `ha_release` will need an update before merging this one in.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/device_tracker.bt_smarthub.markdown [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+ha_release: "0.80"
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.bt_smarthub.markdown]: 0.82

File: source/_components/device_tracker.bt_smarthub.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_components/device_tracker.bt_smarthub.markdown:14]: Please add `ha_iot_class`.

**Comment by frenck** [comment]
@jxwolstenholme Could you please update the requested changes by @MartinHjelmare ? 👍

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**